vulture_os/toolkit/api_parser/hornetsecurity/hornetsecurity.py

```diff
@@ -0,0 +1,361 @@
+
+            # subject
+            subject = log.get('subject')
+            log['is_subject_encoded'] = False
+            if isinstance(subject, str) and not is_ascii(subject):
+                try:
+                    log['subject'] = str(b64encode(subject.encode('utf-8')))
+                    log['is_subject_encoded'] = True
+                except:
+                    log['subject'] = ""
+
+            # destination_hostname
+            destination_hostname : str = log.get('destination_hostname')
+            if destination_hostname and "[" in destination_hostname:
+                log['destination_hostname'] = destination_hostname.split('[')[0]
+            else:
+                log['destination_hostname'] = ""
+
+        ## AUDITING
+        elif kind == "auditing":
+            # new_values
+            new_values = log.get('new_values')
+            if new_values:
+                new_values = loads(new_values)
+                if not isinstance(new_values, dict):
+                    log['new_values'] = {}
+                else:
+                    subject = new_values.get('subject')
+                    if isinstance(subject, str) and not is_ascii(subject):
+                        new_values['is_subject_encoded'] = False
+                        try:
+                            new_values['subject'] = str(b64encode(subject.encode('utf-8')))
+                            new_values['is_subject_encoded'] = True
+                        except:
+                            new_values['subject'] = ""
+                    log['new_values'] = new_values
+
+            # old_values
+            old_values = log.get('old_values')
+            if old_values:
+                old_values = loads(old_values)
+                if not isinstance(old_values, dict):
+                    log['old_values'] = {}
+                else:
+                    subject = old_values.get('subject')
+                    if isinstance(subject, str) and not is_ascii(subject):
+                        old_values['is_subject_encoded'] = False
+                        try:
+                            old_values['subject'] = str(b64encode(subject.encode('urd-8')))
+                            old_values['is_subject_encoded'] = True
+                        except:
+                            old_values['subject'] = ""
+                    log['old_values'] = old_values
+
+        return dumps(log)
+
+
+    def execute(self):
+        for kind in ["auditing", "emails"]:
+            logs = []
+
+            since = self.last_collected_timestamps.get(f"hornetsecurity_{kind}") or (timezone.now() - timedelta(days=7))
+            to = min(timezone.now() - timedelta(minutes=3), since + timedelta(days=1))
+
+            new_logs, new_to = self.get_logs(kind, since, to)
+
+            logger.info(f"[{__parser__}][execute]: Successfully get {len(new_logs)} {kind}'s logs from {since} to {new_to}", extra={'frontend': str(self.frontend)})
```

Comment:
```suggestion
            logger.info(f"[{__parser__}][execute]: Successfully got {len(new_logs)} {kind}'s logs from {since} to {new_to}", extra={'frontend': str(self.frontend)})
```